main/mainTemplate.json

```diff
@@ -0,0 +1,372 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "artifactsBaseUrl": {
+      "type": "string",
+      "metadata": {
+        "artifactsBaseUrl": "/",
+        "description": "The base URL for dependent assets."
+      },
+      "defaultValue": "https://raw.githubusercontent.com/amarozinfoblox/azure-templates/master/main/"
```

Comment:
do we need to refer infobloxopen repo here ?

Reply:
Changed to correct url back.